contents/blog/introducing-hogql.md

```diff
@@ -0,0 +1,111 @@
+
+Speaking of bills, these features are free for all users while in public beta - all we ask is that you [share any feedback with us](http://app.posthog.com/home#supportModal). 
+
+## What is HogQL?
+HogQL is basically our take on SQL. 
+
+Less basically, it's a transition layer over ClickHouse SQL that we've created to empower a wide range of users, from technical product managers who need multi-property breakdowns, to engineers who need to dissect data in more nuanced ways. It's intuitive for teams who are familiar with SQL, but offers a few extra advantages and features.
+
+Unique features to HogQL include simplified access to event and person properties, as well as automatically added joins when you query fields with data on a different table, such as `events.person.properties.$browser`.
+
+You can also use subqueries, joins, table expressions, arrays, aggregations, lambdas and a whole host of other neat SQL features, including aggregations.
+
+We recommend checking the docs for a more detailed [explanation of ClickHouse SQL functions which are supported in HogQL](/manual/hogql).
+
+## HogQL breakdowns
+![HogQL breakdowns](../images/blog/array/hog_breakdown.gif)
+
+A frequent request from teams using PostHog has been the ability to breakdown insight results across multiple properties. To accommodate this, we've added the ability to use HogQL expressions within a breakdown. 
+
+In other words, you can now breakdown results however you like - including by multiple properties at once. Want to breakdown new sign-ups by both pricing tier _and_ overall usage? Now you can.
+
+## HogQL filters
+![HogQL filter](../images/blog/array/hog_filters.gif)
+
+Oh, look. You can use HogQL in filters too. Useful for _filtering_ by multiple properties!
+
+## SQL insights
+![PostHog SQL insights](../images/features/hogql/sql.gif)
+
+If you're familiar with SQL, the easiest and most powerful way to leverage it within PostHog is via the new SQL insight type. This gives you direct SQL access to your data in PostHog, so you can create custom tables, visualization and more. 
+
+SQL insights are especially useful for when you need to do exotic or novel analysis that isn't immediately possible with existing insight types. For example, while you can use a retention insight to discover which product features keep users coming back for more, you could use an SQL insight to find which users keep coming back and identify outliers. 
+
+We're incredibly excited to hear your feedback and see what's possible with SQL access in PostHog, and believe it will give engineers a unique edge and better information than they'd be able to get with other product analytics tools.
+
+To give you a sense of what's possible with SQL, here's an example query we use to summarize data from our from [our 404 page](/hedgehogsgalore) and display which countries prefer their pizzas with pineapple on them.
```

Comment:
Sadly we have a new 404 page without the survey. Thus this link is broken, but not in the right way.

Reply:
```suggestion
To give you a sense of what's possible with SQL, here's an example query we use to summarize data from a survey we ran to determine which countries prefer pineapple on a pizza.
```